runtime/src/common/sgx/avr.rs

```diff
@@ -15,8 +12,9 @@ use serde_json;
 use untrusted;
 use webpki;
 
-#[cfg(target_env = "sgx")]
-use sgx_isa::Report;
+use byteorder::{LittleEndian, ReadBytesExt};
+use sgx_isa::{AttributesFlags, Report};
+use std::io::{Cursor, Read, Seek, SeekFrom};
```

Comment:
now that these aren't conditioned under `target_env`, we can put them into the std -> 3rd party dependencies -> local dependencies order